pkg/apis/hco/v1beta1/hyperconverged_types.go

```diff
@@ -56,25 +57,31 @@ type HyperConvergedConfig struct {
 	NodePlacement *sdkapi.NodePlacement `json:"nodePlacement,omitempty"`
 }
 
-type HyperConvergedFeatureGates map[string]bool
-
-func (fgs HyperConvergedFeatureGates) IsEnabled(fgName string) bool {
-	fg, found := fgs[fgName]
-	return fg && found
+// HyperConvergedFeatureGates is a set of optional feature gates to enable or disable new features that are not enabled
+// by default yet.
+// +optional
+// +k8s:openapi-gen=true
+type HyperConvergedFeatureGates struct {
+	// Allow attaching a data volume to a running VMI; type is boolean; default is false
+	// +optional
+	HotplugVolumes *bool `json:"HotplugVolumes,omitempty"`
```

Comment:
Agree, however, this is the KubeVirt naming, and I prefer to use the exact casing as done there, to avoid confusion.

Reply:
I don't think it'll cause more confusion than non-standard/non-conventional casing in a Kubernetes object schema.
IMO, adhering to the conventions here is of further importance than aligning with an ad-hoc format that is anyway supposed to be removed soon (the configmap), isn't it?
+ we have documentation and schema validation so that users can find their way to the light.